pubspec.yaml

```diff
@@ -114,6 +114,7 @@ flutter:
     - .git/
     - .git/refs/heads/
     - assets/l10n/
+    - assets/id-schema/
```

Comment:
any reasons we have idschema as part of bundle of application?
we are syncing it from server, so if its for reference we can move it out

Reply:
It was added for testing some changes. Removed now.